Backend/Shared.php

```diff
@@ -314,24 +314,31 @@ public function run ( ) {
      * Start the shared worker.
      *
      * @access  public
-     * @param   string  $socket        Socket URI to PHP-FPM server.
-     * @param   string  $workerPath    Path to the shared worker program.
+     * @param   string  $socket             Socket URI to PHP-FPM server.
+     * @param   string  $workerPath         Path to the shared worker program.
+     * @param   array   $fastcgiParameters  Array of additional parameters for FastCGI.
+     * @throw  \Hoa\Worker\Backend\Exception
      * @return  bool
      */
-    public static function start ( $socket, $workerPath ) {
+    public static function start ( $socket, $workerPath, array $fastcgiParameters = array() ) {
 
         $server = new \Hoa\Fastcgi\Responder(
             new \Hoa\Socket\Client($socket)
         );
 
-        return $server->send(array(
+        $headers = [
             'GATEWAY_INTERFACE' => 'FastCGI/1.0',
             'SERVER_PROTOCOL'   => 'HTTP/1.1',
-            'REQUEST_METHOD'    => 'GET',
             'REQUEST_URI'       => $workerPath,
             'SCRIPT_FILENAME'   => $workerPath,
             'SCRIPT_NAME'       => DS . dirname($workerPath)
-        ));
+        ];
+
+        $fastcgiParameters = array_merge([
+            'REQUEST_METHOD'    => 'GET'
+        ], $fastcgiParameters);
+
+        return $server->send(array_merge($fastcgiParameters, $headers));
```

Comment:
you right, we can do `array_merge(['REQUEST_METHOD' => 'GET'], $fastcgiParameters, $headers)`